sklearn/decomposition/tests/test_pca.py

```diff
@@ -454,6 +454,8 @@ def test_pca_zero_noise_variance_edge_cases(svd_solver):
 
     pca.fit(X.T)
     assert pca.noise_variance_ == 0
+    # ensure pca.score works for n_components == n_samples < n_features
+    pca.score(X.T)
```

Comment:
```suggestion
    # Ensure no zero division error for n_components == n_samples < n_features
    pca.score(X.T)
```

Using a more descriptive comment could be helpful, as otherwise it may be unclear what's actually being tested by calling `.score()`.